DROD/RoomScreen.h

```diff
@@ -73,6 +73,7 @@ class CRoomScreen : public CDrodScreen
 	void     ShowScroll() {this->bIsScrollVisible = true; PaintScroll();}
 	void     OpenNoticesBox(CRoomWidget *pRoomWidget);
 	void     GrabNewNotices(CRoomWidget *pRoomWidget);
+	const void UpdateNoticesButton();
```

Comment:
I recall reading recently that `const void` does nothing but I can't find it now - can someone who knows C++ better than me confirm/deny?

Reply:
I've never used `const void` before, but we can remove the 'const'.